acceptance-tests/README.md

```diff
@@ -0,0 +1,115 @@
+Acceptance tests for Blue Ocean
```

Comment:
This needs some editing so it is now correct for this structure (it is mostly ok though)